compiler-rt/lib/tsan/rtl/tsan_interceptors_posix.cpp

```diff
@@ -341,11 +330,6 @@ void ScopedInterceptor::DisableIgnoresImpl() {
 }
 
 #define TSAN_INTERCEPT(func) INTERCEPT_FUNCTION(func)
-#if SANITIZER_FREEBSD || SANITIZER_NETBSD
-#  define TSAN_INTERCEPT_VER(func, ver) INTERCEPT_FUNCTION(func)
-#else
-#  define TSAN_INTERCEPT_VER(func, ver) INTERCEPT_FUNCTION_VER(func, ver)
-#endif
```

Comment:
Why can these be deleted? Where else would TSAN_INTERCEPT_VER be defined?

Reply:
Aren't they still used in lines 3034-3039 of this file?

Reply:
You are right. Fixed:)